gmt/utils.py

```diff
@@ -180,3 +183,64 @@ def is_nonstr_iter(value):
     except TypeError:
         is_iterable = False
     return bool(not isinstance(value, str) and is_iterable)
+
+
+class GMTTempFile():
+    """
+    Context manager for temporaray file.
```

Comment:
temporaray = temporary

Maybe "Context manager for creating closed temporary files" would be better.